openslides/users/rest_auth.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.authentication import BaseAuthentication
+
+from openslides.config.api import config
+
+
+# TODO: Move this into the file 'auth' after fixing circular imports
```

Comment:
1. Some file: import rest_framework.something. -> Load rest_framework_config
2. rest_framework: load config (load openslides.users.auth)
3. openslides.users.auth: import .models
4. openslides.users.models: import openslides.utils.rest_api
5. openslides.utils.rest_api: import rest_framework.something

Step 5 fails, because the config of the rest framework is not loaded yet.